index.html

```diff
@@ -1179,7 +1179,7 @@ <h1>Data Model</h1>
       <tbody>
         <tr>
           <td>
-<a data-cite="INFRA#maps">ordered&nbsp;map</a>
+<a data-cite="INFRA#maps">map</a>
           </td>
           <td>
 A finite ordered sequence of key/value pairs, with no key appearing twice as
```

Comment:
The note saying "Sometimes referred to as just "map" should be now removed or, and that is possibly better, replace with something like "Sometimes referred to as 'ordered map' in the INFRA standard"

Reply:
(Unfortunately, github did not allow me to point at the right line, it only allows me to comment on the changes...)

Reply:
Fixed in f7f247fddece1403999305c0b923af2625617f85.